prediction_market_agent_tooling/markets/omen/omen_contracts.py

```diff
@@ -72,6 +72,29 @@ def getConditionId(
         )
         return id_
 
+    def redeemPositions(
+        self,
+        from_address: ChecksumAddress,
+        from_private_key: PrivateKey,
+        collateral_token_address: str,
+        condition_id: str,
+        parent_collection_id: str,
```

Comment:
```suggestion
        parent_collection_id: HexStr,
```

`build_parent_collection_id` returns `HexStr`, so I guess this should be HexStr too?